man/create.scatterplot.Rd

```diff
@@ -900,6 +900,7 @@ create.scatterplot(
     yaxis.fontface = 1,
     xlab.cex = 1.5,
     ylab.cex = 1.5,
+    # To plot ROC curve, add "s" or "S" to type vector
```

Comment:
The `type` description specifies `s` but not `S`. If both are supported, mention that the characters are case-insensitive.